components/Item/ItemMenu.vue

```diff
@@ -56,9 +63,42 @@ export default Vue.extend({
             }
           });
         }
+
+        if (
+          this.$auth.user?.Policy?.IsAdministrator &&
+          (this.item.Type === 'Folder' || this.item.Type === 'CollectionFolder')
+        ) {
+          menuItems.push({
+            title: this.$t('refreshLibrary'),
+            action: async () => {
+              try {
+                await this.$api.itemRefresh.post({
+                  itemId: this.item.Id,
+                  replaceAllImages: false,
+                  replaceAllMetadata: false
+                });
+                this.pushSnackbarMessage({
```

Comment:
Not sure how necessary this is. Maybe as a "normal" color one, not a success one ?

Reply:
Updated to use the default color 👍